puppeteer-login-script.js

```diff
@@ -48,12 +48,14 @@ const login = async function(context, page) {
       page.click('#submit-login'),
     ]);
 
+    await page.waitForSelector('#username', {timeout: 50000});
     await page.type('#username', 'username1');
+    await page.waitForSelector('#terms-checkbox', {timeout: 50000});
     await page.click('#terms-checkbox');
 
     await Promise.all([
       page.waitForNavigation(),
-      await page.waitForSelector('#signup-submit'),
+      await page.waitForSelector('#signup-submit', {timeout: 50000}),
```

Comment:
Is this waitForSelector necessary here? I thought that page.waitForNavigation would be sufficient.

https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pagewaitfornavigationoptions